src/manifolds/VectorBundle.jl

```diff
@@ -169,6 +201,55 @@ struct VectorBundleBasisData{BBasis<:CachedBasis,TBasis<:CachedBasis}
     vec_basis::TBasis
 end
 
+@doc raw"""
```

Comment:
We could add a comment that explains the product in the name? Something like that it works element wise?

Reply:
Yes, the idea is to apply the retraction, inverse retraction or vector transport element-wise, like on a product manifold. I will add a comment about it.

Reply:
I've added an explanation.

Reply:
That is exactly what helps here, great :)